Production/test/condorSub/dict_Summer20UL16APV_tt.py

```diff
@@ -1,6 +1,19 @@
 flist = {
-    "scenario": "Summer20UL16APV",
-    "samples": [
-        ['Summer20UL16APV.TTToHadronic_TuneCP5_13TeV-powheg-pythia8'],
```

Comment:
seems like you lost `TTToHadronic` in all of these

Reply:
Based on a comment from Alexx, anything with tt + X now lives in the `_tt.py` while the three decay modes for "pure" tt live in `_ttbar.py`.

Reply:
Ah okay, I had missed this point.